<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>40</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;Comic Sans MS&quot;;
color: rgb(247, 247, 247);
background-color: rgb(204, 180, 255);
selection-color: rgb(204, 180, 255);</string>
   </property>
   <property name="text">
    <string>START</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>40</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;Comic Sans MS&quot;;
color: rgb(247, 247, 247);
background-color: rgb(187, 221, 171);
selection-background-color: rgb(187, 221, 171);</string>
   </property>
   <property name="text">
    <string> SCORE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>100</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;Comic Sans MS&quot;;
color: rgb(247, 247, 247);
background-color: rgb(240, 212, 70);
selection-color: rgb(240,212,70);</string>
   </property>
   <property name="text">
    <string>CLEAR</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>100</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 8pt &quot;Comic Sans MS&quot;;
color: rgb(247, 247, 247);
background-color: rgb(251, 194, 255);
selection-color: rgb(251,194,255);</string>
   </property>
   <property name="text">
    <string>CONTINUE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>40</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;Comic Sans MS&quot;;
color: rgb(247, 247, 247);
background-color: rgb(255, 223, 157);
selection-color: rgb(255, 223, 157);</string>
   </property>
   <property name="text">
    <string>PAUSE</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
